### DISCUSSION TITLE
Yolov5 stops training due to patience=30 reached.

### DISCUSSION THREAD

**Discussion by Lio320** [ultralytics/yolov5#4746] [Q&A]
Hi, I tried to train Yolov5 on my custom dataset, everything works fine, but the model stops training after 70 epochs due to the max patience reached (patience = 30). How can I disable it? Since I want to try training for a longer time. I searched in all files for something like early stopping but I can't find anything. I am training using wandb, don't know if it can influence something.

**Comment by glenn-jocher** [comment]
@Lio320 default patience in latest master code is 100 epochs, updated recently in https://github.com/ultralytics/yolov5/pull/4679. You define a new `--patience` with:
```shell
python train.py --patience 200
```
